.expeditor/scripts/habitat-test.ps1

```diff
@@ -0,0 +1,10 @@
+$ErrorActionPreference = 'Stop'
+
+$EXPEDITOR_PKG_IDENTS_CHEFINFRACLIENTX86_64WINDOWS = 'chef/chef-infra-client/18.0.179/20221109104144'
```

Comment:
Any narration via comment you can do to help this be more supportable? Why a hardcoded version, for example?

Reply:
I actually don't know what sets that. It's hard-coded in the copy sitting in Main too. I suspect some Expeditor work item is setting that and turning this loose to run. I'll investigate this so we understand it. Hard-coded is bad